src/Core/AspectKernel.php

```diff
@@ -190,11 +218,13 @@ protected function getDefaultOptions()
             'debug'     => false,
             'appDir'    => __DIR__ . '/../../../../../../',
             'cacheDir'  => null,
+            'cachePerms'=> null,
             'features' => $features,
 
-            'includePaths'       => array(),
-            'excludePaths'       => array(),
-            'containerClass'     => static::$containerClass,
+            'includePaths'           => array(),
+            'excludePaths'           => array(),
+            'containerClass'         => static::$containerClass,
+            'cachePathResolverClass' => static::$cachePathResolverClass,
```

Comment:
This options is not needed. Customization is possible via container class extension. You can define a parameter in the container class with default value, then extend this class in your application and configure a kernel to your specific container class with additional services/overridden parameters.